packages/amplify-nodejs-function-runtime-provider/src/utils/invoke.ts

```diff
@@ -20,11 +20,15 @@ export function invoke(options: InvokeOptions): Promise<any> {
           console.log(logs);
```

Comment:
This will catch lambdas which don't return a response, could we also add a block here to reject when an error is throw inside the lambda?
```js
lambdaFn.catch((err) => {
  reject(err.message);
});
```